src/utils/utils.ts

```diff
@@ -145,6 +147,39 @@ export class Utils {
     return true;
   }
 
+  public static async getTarget(): Promise<ITarget> {
+    return await cfGetTarget();
+  }
+
+  public static async isLoggedInToCf(): Promise<boolean> {
+    try {
+      const target: ITarget = await this.getTarget();
+      // user is connected to CF
+      if (
+        !isEmpty(target.user) &&
+        !isEmpty(target.org) &&
+        !isEmpty(target.space) &&
+        !isEmpty(target["api endpoint"])
```

Comment:
redundant. if user attribute is fulfilled the endpoint must exist

Reply:
Done